src/platform/sdl/main.c

```diff
@@ -38,11 +38,20 @@
 #include <signal.h>
 
 #define PORT "sdl"
+#define MAX_LOG_BUF 1024
 
 static void mSDLDeinit(struct mSDLRenderer* renderer);
 
 static int mSDLRun(struct mSDLRenderer* renderer, struct mArguments* args);
 
+void setLogger(struct mCore* core);
```

Comment:
This function should be static (and as such probably prefixed with an _).

Reply:
Fixed in https://github.com/mgba-emu/mgba/pull/2345/commits/fac41951f3984e0395765df6c02f0fb02c0ab800 .